external/flux/flux-examples/README.md

```diff
@@ -0,0 +1,68 @@
+# Flux Examples
+A collection of examples illustrating various capabilities.
+
+## Building From Source and Running
```

Comment:
It should be changed to clone Storm project and cd to external/flux, and build. (or build whole modules from Storm root)